src/block/horizontal-scroller/frontend-horizontal-scroller.js

```diff
@@ -11,6 +11,14 @@ class StackableHorizontalScroller {
 		let initialClientX = 0
 
 		els.forEach( el => {
+			// get all <a> tags for inner columns
+			const children = el.querySelectorAll( '.stk-block-link' )
```

Comment:
I just realized that all links would suffer from the problem, so I added `, a` in the selector

```suggestion
			const children = el.querySelectorAll( '.stk-block-link, a' )
```